vterm.el

```diff
@@ -301,40 +305,32 @@ variable `next-error-function', so we should call this function in
 This is the value of `next-error-function' in Compilation buffers."
   (interactive "p")
   (let* ((pt (point))
-         (msg (compilation-next-error (or n 1) nil
-				                      (or compilation-current-error
-					                      compilation-messages-start
-					                      (point-min))))
          (default-directory default-directory)
          (pwd (vterm--get-pwd)))
     (when pwd
       (setq default-directory pwd))
     (goto-char pt)
     (compilation-next-error-function n reset)))
 
-(defmacro vterm-define-key (key)
-  "Define an command sending M-[a-z] or C-[a-z] to vterm."
-  (declare (indent defun)
-           (doc-string 3))
-  `(defun ,(intern(format "vterm-send-%s" key))()
-     ,(format "Sends %s to the libvterm."  key)
-     (interactive)
-     (vterm-send-key ,(char-to-string (get-byte (1- (length key)) key)) nil
-                     ,(string-prefix-p "M-" key)
-                     ,(string-prefix-p "C-" key))))
-
-(defmacro vterm-bind-key (key)
-  (declare (indent defun)
-           (doc-string 3))
-  `(define-key vterm-mode-map (kbd ,key)
-     #',(intern (format "vterm-send-%s"  key))))
-
-(mapc (lambda (key)
-        (eval `(vterm-define-key ,key)))
-      (cl-loop for prefix in '("C-" "M-")
-               append (cl-loop for char from ?a to ?z
-                               for key = (format "%s%c" prefix char)
-                               collect key)))
+
+(eval-and-compile
+  (defmacro vterm-define-key (key)
+    "Define an command sending M-[a-z] or C-[a-z] to vterm."
```

Comment:
Can you please fix the spelling mistake in this docstring?
"an command" -> "a command"

Feel free to expand upon/rewrite the docstring.